modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc

```diff
@@ -7,46 +7,88 @@
 [id="querying-metrics-for-all-projects-as-an-administrator_{context}"]
 = Querying metrics for all projects as a cluster administrator
 
-As a cluster administrator or as a user with view permissions for all projects, you can access metrics for all default {product-title} and user-defined projects in the Metrics UI.
+As a
+ifndef::openshift-dedicated,openshift-rosa[]
+cluster administrator
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+`dedicated-admin`
+endif::openshift-dedicated,openshift-rosa[]
+or as a user with view permissions for all projects, you can access metrics for all default {product-title} and user-defined projects in the Metrics UI.
+
+ifdef::openshift-dedicated,openshift-rosa[]
+[NOTE]
+====
+Only dedicated administrators have access to the third-party UIs provided with {product-title} monitoring.
+====
+endif::openshift-dedicated,openshift-rosa[]
 
 .Prerequisites
 
+ifndef::openshift-dedicated,openshift-rosa[]
 * You have access to the cluster as a user with the `cluster-admin` role or with view permissions for all projects.
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `dedicated-admin` role or with view permissions for all projects.
+endif::openshift-dedicated,openshift-rosa[]
 * You have installed the OpenShift CLI (`oc`).
 
 .Procedure
 
-. Select the *Administrator* perspective in the {product-title} web console.
+. From the *Administrator* perspective in the {product-title} web console, select *Observe* -> *Metrics*.
 
-. Select *Observe* -> *Metrics*.
+. To add one or more queries, do any of the following:
++
+|===
+|To... |Do this...
```

Comment:
https://github.com/openshift/openshift-docs/pull/60762/files#r1214688029

Reply:
I changed the table column headings to "Option" and "Description".